docs/submission/faq.md

```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
 ---
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
+
+## Frequently Asked Questions
+
+1. How do I submit multiple values for a field? For example, I want to submit multiple symptoms for the `presenting_symptoms` field.
+
+   To include multiple values, separate values with a pipe delimiter '|' within your file. For example: `Anemia|Fatigue|Fever`.
+
+2. Is there any resource to refer to in order to follow staging guidelines?
+
+    Please refer to the guidelines provides by the [AJCC Cancer Staging](https://cancerstaging.org/Pages/default.aspx) website. Other helpful resources include the [CAnswer Forum](http://cancerbulletin.facs.org/forums/).
```

Comment:
typo: 
provides to `provided`

Also, change "Other helpful resources include the"... to "Another helpful resource is the..."